.changeset/mean-spiders-design.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/plugin-search-backend': minor
+---
+
+Be less restrictive with unknown keys on query endpoint by adding .passthrough() method to parse the `requestSchema`
+
+```diff
+-      const parseResult = requestSchema.safeParse(req.query);
++      const parseResult = requestSchema.passthrough().safeParse(req.query);
+```
```

Comment:
Hey @pedronastasi!  For context, this changeset entry eventually gets rolled up into [this changelog](https://github.com/backstage/backstage/blob/master/plugins/search-backend/CHANGELOG.md), which is often read by adopters as they upgrade packages.

With that audience in mind: I would recommend removing this diff line; your average adopter might not know/understand what it means, nor would they necessarily need that info.